rootfs/api/models.py

```diff
@@ -475,7 +475,7 @@ def _default_scale(self, user, release):
     def logs(self, log_lines=str(settings.LOG_LINES)):
         """Return aggregated log data for this application."""
         try:
-            url = "http://{}:{}/{}?log_lines={}".format(settings.LOGGER_HOST, settings.LOGGER_PORT,
+            url = "http://{}:{}/{}/?log_lines={}".format(settings.LOGGER_HOST, settings.LOGGER_PORT,
```

Comment:
Yeah I've already made the slash optional but haven't pushed that yet
On Jan 13, 2016 4:45 PM, "krancour" <notifications@github.com> wrote:

> In rootfs/api/models.py
> <https://github.com/deis/workflow/pull/263#discussion_r49669167>:
>
> > @@ -475,7 +475,7 @@ def _default_scale(self, user, release):
> >      def logs(self, log_lines=str(settings.LOG_LINES)):
> >          """Return aggregated log data for this application."""
> >          try:
> > -            url = "http://{}:{}/{}?log_lines={}".format(settings.LOGGER_HOST, settings.LOGGER_PORT,
> > +            url = "http://{}:{}/{}/?log_lines={}".format(settings.LOGGER_HOST, settings.LOGGER_PORT,
>
> This is the spot I thought I was commenting on before. I checked and the
> extra slash isn't needed here. But it also doesn't hurt, so I won't make a
> fuss. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/deis/workflow/pull/263/files#r49669167>.
>